.npmrc

```diff
@@ -0,0 +1,2 @@
+; enable npm install to succeed by forcing the old method of peer dependency handling
+legacy-peer-deps=true
```

Comment:
nice change for letting different versions of npm work. (npm install --legacy-peer-deps will also work for such npm versions)